### PR TITLE
fix: refactor message box

### PR DESCRIPTION
**Pull request by adriawh**
# Syncing Figma, storybook and code

 ### Changes
- Split the `MessageBox` component into `MessageInfoBox` and `MessageInfoText` as it is in Figma.
- Updated storybook
- Updated design system page 
- All occurrences of  `MessageBox` with the `subtle` flag has been replaced with `MessageInfoText`.
- Global messages have kept the `subtle` flag but switches between the two components. As proposed by @gorandalum.
<details open>
<summary>Images</summary>

</details>

### CODE REVIEW

File: src/storybook/stories/MessageInfoText.stories.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  component: MessageInfoText,
+  argTypes: {
+    theme: {
+      control: 'select',
+      options: ['light', 'dark'],
+    },
+  },
+  args: {
+    theme: 'light',
+    message: 'The message body.',
```
Comment (marius-at-atb) [inline src/storybook/stories/MessageInfoText.stories.js:17]: Should messages normally end with a "."?
When the icon is on the right, I think it looks better without

**Review by marius-at-atb — APPROVED** [review]
Excellent!

File: ios/Podfile.lock [hunk 2]
```diff
@@ -1907,13 +1907,13 @@ SPEC CHECKSUMS:
...
-  TokenStateReactNativeLib: 0f6875d9d039385a07456588701f638dc5463a79
+  TokenStateReactNativeLib: 30f51acbc60a7064ce4d9cfa580c2553919667f2
   Turf: 13d1a92d969ca0311bbc26e8356cca178ce95da2
   Yoga: 3efc43e0d48686ce2e8c60f99d4e6bd349aff981
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
 
 PODFILE CHECKSUM: 65771909d1de0011499a623efd21303801625f9f
 
-COCOAPODS: 1.13.0
+COCOAPODS: 1.12.1
```
[the remark this answers is not in the document]
Comment (adriawh) [reply ios/Podfile.lock:1919]: Yes, it got in there by accident. Will remove it

**Review by gorandalum — COMMENTED** [review]
The implementation looks good. But the message type "general" should be either be removed from Figma or implemented in the code, the keep them in sync. I'm guessing removed from Figma, since it is not in use.

**Pull request merged**